cmake/std/atdm/utils/get_known_system_info.sh

```diff
@@ -11,127 +11,196 @@ unset ATDM_CONFIG_CDASH_HOSTNAME
 unset ATDM_CONFIG_SYSTEM_NAME
 unset ATDM_CONFIG_SYSTEM_DIR
 
-# Assert this script is sourced, not run!
 called=$_
 if [ "$called" == "$0" ] ; then
   echo "This script '$0' is being called.  Instead, it must be sourced!"
   exit 1
 fi
 unset called
 
-# Assert that ATDM_CONFIG_BUILD_NAME is set!
 if [ -z "$ATDM_CONFIG_BUILD_NAME" ] ; then
   echo "Error, must set ATDM_CONFIG_BUILD_NAME in env!"
   return
 fi
 
-export ATDM_CONFIG_REAL_HOSTNAME=`hostname`
-#echo "Hostname = '$ATDM_CONFIG_REAL_HOSTNAME'"
+if [ -z "$ATDM_CONFIG_SCRIPT_DIR" ] ; then
+  echo "Error, must set ATDM_CONFIG_SCRIPT_DIR in env!"
+  return
+fi
 
-ATDM_HOSTNAME=
-ATDM_SYSTEM_NAME=
+source ${ATDM_CONFIG_SCRIPT_DIR}/utils/get_system_info_utils.sh
 
-ATDM_IS_CEE_RHEL6_MACHINE=
-
-# Specifically named systems
-if [[ $ATDM_CONFIG_REAL_HOSTNAME == "hansen"* ]] ; then
-  ATDM_HOSTNAME=hansen
-  ATDM_SYSTEM_NAME=shiller
-elif [[ $ATDM_CONFIG_REAL_HOSTNAME == "shiller"* ]] ; then
-  ATDM_HOSTNAME=shiller
-  ATDM_SYSTEM_NAME=shiller
-elif [[ $ATDM_CONFIG_REAL_HOSTNAME == "white"* ]] ; then
-  ATDM_HOSTNAME=white
-  ATDM_SYSTEM_NAME=ride
-elif [[ $ATDM_CONFIG_REAL_HOSTNAME == "ride"* ]] ; then
-  ATDM_HOSTNAME=ride
-  ATDM_SYSTEM_NAME=ride
-elif [[ $ATDM_CONFIG_REAL_HOSTNAME == "mutrino"* ]] ; then
-  ATDM_HOSTNAME=mutrino
-  ATDM_SYSTEM_NAME=mutrino
-elif [[ $ATDM_CONFIG_REAL_HOSTNAME == "waterman"* ]] ; then
-  ATDM_HOSTNAME=waterman
-  ATDM_SYSTEM_NAME=waterman
-
-# cts1 systems
-elif [[ $ATDM_CONFIG_REAL_HOSTNAME == "serrano"* ]] \
-  || [[ $ATDM_CONFIG_REAL_HOSTNAME =~ ser[0-9]+ ]] ; then
-  ATDM_HOSTNAME=serrano
-  ATDM_SYSTEM_NAME=serrano
-elif [[ $ATDM_CONFIG_REAL_HOSTNAME == "eclipse"* ]] \
-  || [[ $ATDM_CONFIG_REAL_HOSTNAME =~ ec[0-9]+ ]] ; then
-  ATDM_HOSTNAME=eclipse
-  ATDM_SYSTEM_NAME=serrano
-elif [[ $ATDM_CONFIG_REAL_HOSTNAME == "ghost"* ]] \
-  || [[ $ATDM_CONFIG_REAL_HOSTNAME =~ gho[0-9]+ ]] ; then
-  ATDM_HOSTNAME=ghost
-  ATDM_SYSTEM_NAME=serrano
-
-# tlcc2 systems
-elif [[ $SNLSYSTEM == "tlcc2"* ]] ; then
-  ATDM_SYSTEM_NAME=tlcc2
-  if [[ $SNLCLUSTER == "" ]] ; then
-    ATDM_HOSTNAME=$ATDM_CONFIG_REAL_HOSTNAME
-  else
-    ATDM_HOSTNAME=$SNLCLUSTER
-  fi
+export realHostname=`hostname`
```

Comment:
Is `realHostname` used outside of `get_known_system_info.sh`? If not, I would consider removing `export` here.

Reply:
That was a good catch.  Actually, this was renamed from the var `ATDM_CONFIG_REAL_HOSTNAME` which **is** used outside of this script and therefore needs to be exported (which I would have seen if I would have read my own documentation).  Thanks for catching this!